### PR TITLE
asuite: Add version 2.0.0

### PR DESCRIPTION
**Pull request by salvadorbs**

### CODE REVIEW

File: bucket/ASuite.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+   "autoupdate":{
+      "architecture":{
+         "64bit":{
+            "url":"https://github.com/salvadorbs/Asuite/releases/download/$version/asuite-$version.7z"
+         },
+         "32bit":{
+            "url":"https://github.com/salvadorbs/Asuite/releases/download/$version/asuite-x64-$version.7z"
+         }
+      }
+   }
```
Comment (Ash258) [inline bucket/ASuite.json:46]:
```suggestion
    "version": "2.0.0",
    "description": "Portable applications launcher",
    "homepage": "http://www.salvadorsoftware.com/asuite",
    "license": "GPL-3.0-only",
    "architecture": {
        "64bit": {
            "url": "https://github.com/salvadorbs/Asuite/releases/download/v2.0.0/asuite_x64-v2.0.0.7z",
            "hash": "1366d2325e48d775f772959abd2014bd47b1fdc780033dcffe7b0d5e6712c23a",
            "bin": [
                "asuite_x64.exe",
                [
                    "asuite_x64.exe",
                    "asuite"
                ]
            ],
            "shortcuts": [
                [
                    "asuite_x64.exe",
                    "ASuite"
                ]
            ]
        },
        "32bit": {
            "url": "https://github.com/salvadorbs/Asuite/releases/download/v2.0.0/asuite-v2.0.0.7z",
            "hash": "ac450a8722ac4c5fa20482d732150e55c862da49c9d3994ddd01258460ab824f",
            "bin": "asuite.exe",
            "shortcuts": [
                [
                    "asuite.exe",
                    "ASuite"
                ]
            ]
        }
    },
    "pre_install": "'asuite_x64.sqlite', 'asuite.sqlite' | ForEach-Object { if (-not (Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null } }",
    "persist": [
        "backup",
        "cache",
        "asuite_x64.sqlite",
        "asuite.sqlite"
    ],
    "checkver": {
        "github": "https://github.com/salvadorbs/Asuite"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/salvadorbs/Asuite/releases/download/v$version/asuite_x64-v$version.7z"
            },
            "32bit": {
                "url": "https://github.com/salvadorbs/Asuite/releases/download/v$version/asuite-v$version.7z"
            }
        }
    }
```

**Comment by Ash258** [comment]
Why do you request the review when you did not complete the first one??

image

**Comment by salvadorbs** [comment]
I forgot to fix persist settings. For now, I close this pull and I will open when I fix it. Thanks.

**Comment by Ash258** [comment]
I fixed the persist in the suggestion

**Comment by salvadorbs** [comment]
Sorry. I didn't see your fixes. I renamed manifest file and reopen pull request, but if you want a open another pull request with a better and cleaner git history. Thanks.

**Pull request merged**